src/main/java/com/mongodb/jdbc/NoCheckStateJsonWriter.java

```diff
@@ -0,0 +1,17 @@
+package com.mongodb.jdbc;
+
+import java.io.Writer;
+import org.bson.json.JsonWriter;
+import org.bson.json.JsonWriterSettings;
+
+public class NoCheckStateJsonWriter extends JsonWriter {
```

Comment:
Maybe we could add a comment on this one.
Something like this maybe :
JsonWriter which allow to write any Json value without validating it is constructing a valid document.

Reply:
Added a comment 👍